src/validations/index.js

```diff
@@ -20,7 +20,9 @@ function runValidation (validation, param, pkgInfo, errs) {
         .run(param, pkgInfo)
         .then(() => done(true))
         .catch(err => {
-            errs.push(err);
+            (err && err.length && err.length > 0 && err.push)
```

Comment:
You can check for an array using `Array.isArray(err)`

Reply:
Done.
Now the validation errors for vulnerabilities should look like:

```console
Running validations
-------------------

✖ Checking for the vulnerable dependencies
✖ Checking for the uncommitted changes
✓ Checking for the untracked files
✓ Checking for the sensitive data in the working tree
✓ Validating branch
✓ Validating git tag

ERRORS
  * Vulnerability found in publish-please@2.4.1
        inside publish-please@2.4.1 -> ban-sensitive-files@1.9.2 -> ggit@2.4.2 -> lodash@4.17.4
        Update to version 4.17.5 or later.
        Advisory: https://nodesecurity.io/advisories/577
  * Vulnerability found in publish-please@2.4.1
        inside publish-please@2.4.1 -> nsp@2.8.1 -> https-proxy-agent@1.0.0
        Update to version 2.2.0 or later.
        Advisory: https://nodesecurity.io/advisories/593
  * Vulnerability found in publish-please@2.4.1
        inside publish-please@2.4.1 -> nsp@2.8.1 -> joi@6.10.1 -> hoek@2.16.3
        Update to version 4.2.1, 5.0.3 or later.
        Advisory: https://nodesecurity.io/advisories/566
  * Vulnerability found in testcafe@0.20.0-alpha.4
        inside testcafe@0.20.0-alpha.4 -> testcafe-hammerhead@13.4.4 -> css@2.2.1 -> source-map-resolve@0.3.1 -> atob@1.1.3
        Update to version 2.1.0 or later.
        Advisory: https://nodesecurity.io/advisories/646
  * Vulnerability found in lodash@4.16.4
        Update to version 4.17.5 or later.
        Advisory: https://nodesecurity.io/advisories/577
  * Vulnerability found in ms@0.7.0
        Update to version 0.7.1 or later.
        Alternatively, apply a reasonable length limit to parsed version strings.
        Advisory: https://nodesecurity.io/advisories/46
  * There are uncommitted changes in the working tree.
```

This report corresponds to the following package.json content:
```json
"dependencies": {
    "@types/minimist": "1.2.0",
    "@types/node": "9.6.1",
    "@types/semver": "5.5.0",
    "callsite-record": "4.1.1",
    "chalk": "2.4.1",
    "minimist": "1.2.0",
    "rimraf": "2.6.2",
    "semver": "5.5.0",
    "testcafe": "0.20.0-alpha.4",
    "ts-node": "6.0.2",
    "typescript": "2.8.3",
    "lodash": "4.16.4",
    "ms": "0.7.0",
    "publish-please": "2.4.1"
  },
````